### DISCUSSION TITLE
2021-06-09 Weekly Meeting 7 Agenda / Discussion

### DISCUSSION THREAD

**Discussion by robertlong** [omigroup/omigroup#43] [Weekly Meetings]
Weekly Meeting 7 is happening on Wednesday 2021-06-09 from 9:00 AM PDT - 10:00 AM PDT

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR Discord](https://discord.gg/NJtT9grz5E).

## Agenda
 - [W3C Community Group Charter](https://github.com/omigroup/OMI/pull/45)
 - [OMI Community Virtual Space Design](https://github.com/omigroup/OMI/issues/44)
 -  OpenCollective transparent funding [about opencollective](https://opencollective.com/opensource)
 - [Virtual World URIs](https://github.com/omigroup/OMI/discussions/23)
 - [OMI Identity](https://github.com/omigroup/OMI/issues/36)

 
 Make sure to comment below with links to your proposals so we can get time scheduled to talk about them.

**Comment by mrmetaverse** [comment]
Evo proposed that we chat about this at next weeks meeting: 
_Here's a tool & structure to discuss for next week: https://opencollective.com/opensource as we get organized, orienting toward transparency in all activities_

**Comment by robertlong** [comment]
Notes from the meeting: https://hackmd.io/@robertlong/omi-meeting-7
